stl/src/xtime.cpp

```diff
@@ -46,7 +46,8 @@ namespace {
 
 _EXTERN_C
 
-_CRTIMP2_PURE long long __cdecl _Xtime_get_ticks() { // get system time in 100-nanosecond intervals since the epoch
+_CRTIMP2_PURE long long __cdecl _Xtime_get_ticks() noexcept {
+    // get system time in 100-nanosecond intervals since the epoch

```

Comment:
It's been hard work causing all the merge conflicts on my own since @BillyONeal left the team.